tests/cephadm/test_osd_service_operations.py

```diff
@@ -0,0 +1,66 @@
+from cli.utilities.operations import wait_for_cluster_health
+from cli.utilities.utils import get_service_id, get_service_state, set_service_state
+from utility.log import Log
+
+
+class CephServiceFailError(Exception):
+    pass
+
+
+class CephServiceStartError(Exception):
+    pass
+
+
+class CephClusterHealthError(Exception):
```

Comment:
Replace this with - https://github.com/red-hat-storage/cephci/blob/master/cli/exceptions.py#L26